src/Pokemon.java

```diff
@@ -0,0 +1,104 @@
+package src;
+
+public abstract class Pokemon {
+    // Variables
+    private String name;
+    private double height;
+    private double weight;
+    private String sound;
+    private String food;
+    private String ability;
+    private String weakness;
+
+
+    // Constructor
+    public Pokemon(String name, double height, double weight, String sound, String food, String ability, String weakness) {
+        this.name = name;
+        this.height = height;
+        this.weight = weight;
+        this.sound = sound;
+        this.food = food;
+        this.ability = ability;
+        this.weakness = weakness;
+    }
+
+
+    // Methods
```

Comment:
Je kan je methode het beste een naam geven die aangeeft wat hij doet, In dit geval printWeight